cmd/main.go

```diff
@@ -21,6 +21,7 @@ func main() {
 		Name:        "mystery",
 		Usage:       "generate mystery ingredients for your next project",
 		Description: "Inspired by the Food Network show Chopped, mystery provides you with mystery ingredients to help kickstart your new or existing project.",
+		Version: 	 "v0.1.0",
```

Comment:
This is good, but we’ll want to take it in as a build arg so that we don’t have to change source every version. Do you know how to do that?